repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/admin/IdentityServiceAdminConsoleAuthenticator.java

```diff
@@ -226,7 +225,7 @@ private Set<String> getScopes(ClientRegistration clientRegistration)
     private Set<String> getSupportedScopes(Scope scopes)
     {
         return scopes.stream()
-            .filter(scope -> SCOPES.contains(scope.getValue()))
+            .filter(scope -> identityServiceConfig.getAdminConsoleScopes().contains(scope.getValue()))
```

Comment:
Maybe we could introduce method `hasAdminConsoleScope` and then use method reference in this filter?